common/app/controllers/EmailSignupController.scala

```diff
@@ -110,7 +110,7 @@ class EmailSignupController(
 
         identityNewsletter match {
           case Some(newsletter) =>
-            Cached(1.day)(
+            Cached(1.hour)(
```

Comment:
What's the best way to test and verify this, just in case there's something in another repo that means that we need to change it in two places?

Reply:
We should be able to see that change on the HTTP headers of these requests